marquez_airflow/utils.py

```diff
@@ -140,3 +142,9 @@ def add_airflow_info_to(task, steps_metadata):
             MARQUEZ_AIRFLOW_VERSION
 
     return steps_metadata
+
+
+def handle_fail(cause):
```

Comment:
Linking a comment I left in `marquez-python` that's similar: https://github.com/MarquezProject/marquez-python/pull/116/files#r508815928

Reply:
refer to my reply at that link.

Reply:
done.